### PR TITLE
Add Dijkstra paper which defines GC abstractions and more GC performance papers

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2096,3 +2096,37 @@ @inproceedings{blackburn2004myths
...
+    number = {11},
+    month = Nov,
+    year = {1970},
+    pages = {677--678},
+    publisher = {ACM},
+}
+
+@inproceedings{shahriyar2012down,
+    author = {Shahriyar, Rifat and Blackburn, Stephen M. and Frampton, Daniel},
+    title = {Down for the Count?  Getting Reference Counting Back in the Ring},
```
Comment (ltratt) [inline softdev.bib]: Double space after `?`.

File: softdev.bib [hunk 2]
```diff
@@ -2096,3 +2096,37 @@ @inproceedings{blackburn2004myths
...
+    booktitle = {ISMM},
+    series = {ISMM '12},
```
Comment (ltratt) [inline softdev.bib]: THis field isn't needed.

File: softdev.bib [hunk 3]
```diff
@@ -2096,3 +2096,37 @@ @inproceedings{blackburn2004myths
...
+    year = {2012},
+    pages = {73--84},
+    month = Jun,
+    publisher = {ACM},
+}
+
+@article{dijkstra1978on,
```
Comment (ltratt) [inline softdev.bib]: I suspect ol' Edsger wrote a lot of articles starting "on". Maybe `dijkstra1978gc` or something like that?

File: softdev.bib (hunk 1 above)
Comment (jacob-hughes) [reply softdev.bib]: Fixed.

File: softdev.bib (hunk 3 above)
Comment (jacob-hughes) [reply softdev.bib]: Fair enough. Updated

File: softdev.bib (hunk 2 above)
Comment (jacob-hughes) [reply softdev.bib]: Removed `series` field.

**Comment by jacob-hughes** [comment]
Addressed comments.

**Comment by ltratt** [comment]
Please squash.

**Comment by jacob-hughes** [comment]
Squashed!

**Pull request merged**